### DISCUSSION TITLE
Forgotten temple expired warning

### DISCUSSION THREAD

**Discussion by LA-son** [mainIine/foe-helfer-extension#2739] [Ideas]
I often forget about The Forgotten Temple expiring, and lose a bunch of FPs.
If possible, please add a warning when Forgotten Temple expired, same as the warning in GBG when only rogues present. The warning can be turned off in setting.
If there's already a feature like this, please tell me how to use it, thanks.

**Comment by outoftheline** [comment]
yes there is a warning for this, it's a general alert similar to GBG ones:
![image](https://github.com/mainIine/foe-helfer-extension/assets/7482863/afc5538d-c8b7-468c-b617-79209f1f492f)

you might need to check if you allow your browser to send alerts in you OS settings.
